rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :book, :person
+  attr_reader :date
```

Comment:
- [ OPTIONAL ] I think the previous code reviewer didn't clarify well, so here data it should be attr_accessor instead of attr_reader because we want to set and get the data not just to get it so kindly update it 🔥💯👍🏻